src/vs/workbench/services/search/common/search.ts

```diff
@@ -419,6 +419,9 @@ export interface ISearchConfigurationProperties {
 			preserveInput: boolean;
 		};
 	};
+	quickOpen: {
+		seedStringFromSelection: boolean;
+	};
```

Comment:
looks like a missed this last time, but I believe you can just remove this since it's already in `IWorkbenchSearchConfigurationProperties` and then just cast the configuration object as a `IWorkbenchSearchConfiguration` like in https://github.com/microsoft/vscode/blob/c72447e8d8aaa7497c9a4bd68bc4301584b92beb/src/vs/workbench/contrib/search/browser/anythingQuickAccess.ts#L194

Reply:
The problem with that is 
IWorkbenchSearchConfiguration is in contrib folder, and because of import rules we cant use it in quickAccessActions.ts

Reply:
can you add a comment with an explanation of why it's here  and maybe link it to the todo that you've added for this https://github.com/microsoft/vscode/pull/197380#discussion_r1382031132?

Reply:
Sure, I can add 
// As IWorkbenchSearchConfiguration is present in the contrib folder, we can not import it in quickAccessActions.ts
// Hence this property is added here again, so that it can be used in the quickAccessActions.ts
I am not sure what you meant by linking it to the todo though